packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -42,70 +28,199 @@ Note that the image toolbar button {@icon @ckeditor/ckeditor5-icons/theme/icons/
+CKBox offers two primary methods for loading the library: using a Content Delivery Network (CDN) for fast, script-based integration, or using a Package Manager (npm/yarn) for modern application bundles.
 
-The easiest way to load the CKBox library is to include the `<script>` tag loading the `ckbox.js` file first:
+### Method 1: Installation from CDN
+
+The easiest way to load the CKBox library is to include the following `<script>` tag in your HTML header that points to the CKBox CDN, ensuring it loads before the rich text editor:
 
 ```html
-<script src="https://cdn.ckbox.io/ckbox/2.4.0/ckbox.js"></script>
+<script src="https://cdn.ckbox.io/ckbox/{%CKBOX_VERSION%}/ckbox.js"></script>
 ```
 
-Please note, that while using the `latest` call instead of a version number is available, it is not advised. The latest version may bring breaking changes that will stall your CKBox integration.
-
-The CKBox feature requires one of the following plugins to be loaded to work correctly:
+When referencing the CKBox library via CDN, you must specify the version number (for example: `{%CKBOX_VERSION%}` or `latest`).
 
-* {@link module:image/imageblock~ImageBlock The `ImageBlock` feature}
-* {@link module:image/imageinline~ImageInline The `ImageInline` feature}
-* {@link module:image/image~Image The `Image` feature} (a glue plugin that loads both the `ImageBlock` and `ImageInline` features)
+<info-box note>
+	While the ability to use the shorthand `latest` tag instead of a specific version number is available, it is strongly advised against. The use of the latest tag may inadvertently introduce breaking changes into your application, which could cause your CKBox integration to stall unexpectedly.
+</info-box>
 
-If you do not have any of these plugins in your editor, install one and add it to your plugin list.
+Suppose you wish to change the default language of the CKBox, which supports over 40 official translations. In that case, you must ensure you load the corresponding translation file after loading the main CKBox library. For example, to load the Spanish translation (es):
 
-Please also remember, that the CKBox plugin requires the following dependency plugins to work properly: `PictureEditing`, `ImageUpload`, and `CloudServices`.
+```html
+<script src="https://cdn.ckbox.io/ckbox/{%CKBOX_VERSION%}/ckbox.js"></script>
+<script src="https://cdn.ckbox.io/ckbox/{%CKBOX_VERSION%}/translations/es.js"></script>
+```
 
-You must include the `CKBoxImageEdit` plugin if you want to use CKBox image editing capabilities from within CKEditor&nbsp;5.
+Once imported, you can use it within your application:
 
-Finally, add {@link module:ckbox/ckbox~CKBox} to your plugin list and toolbar, and [configure](#configuration) the feature as needed. An example configuration may look like this:
+```html
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{%CKBOX_VERSION%}/ckeditor5.css">
```

Comment:
Are we sure this one is ok?

Reply:
Good catch, most probably `{@var ckeditor5-version}` should apply here.